service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -48,13 +42,16 @@ public class MigrateCaseController extends CallbackController {
     private final ManageOrderDocumentScopedFieldsCalculator fieldsCalculator;
 
     private final Map<String, Consumer<CaseDetails>> migrations = Map.of(
-        "DFPL-1144", this::run1144,
-        "DFPL-872rollback", this::run872Rollback,
-        "DFPL-1072", this::run1072,
-        "DFPL-1163", this::run1163,
-        "DFPL-1165", this::run1165,
-        "DFPL-1192", this::run1192,
-        "DFPL-1215", this::run1215
+        "DFPL-1204", this::run1204,
+        "DFPL-1064", this::run1064,
```

Comment:
should be removed? It had been done.

Reply:
Imagine there should be four tests should be added and some of them are removed.